L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc

```diff
@@ -233,8 +248,15 @@ L1FPGATrackProducer::L1FPGATrackProducer(edm::ParameterSet const& iConfig)
     tableTREFile = iConfig.getParameter<edm::FileInPath>("tableTREFile");
   }
 
+  // book ED output token for clock and bit accurate tracks
+  edPutTokenTracks_ = produces<Streams>("Level1TTTracks");
+  // book ED output token for clock and bit accurate stubs
+  edPutTokenStubs_ = produces<StreamsStub>("Level1TTTracks");
   // book ES product
```

Comment:
Comment should make clear this ESProduct determines output channels of Tracklet reco.

Reply:
done